Assets/PatchKit Patcher/Patcher.unity

```diff
@@ -1,19 +1,19 @@
 %YAML 1.1
```

Comment:
Have you made any changes to the scene or is it just the effect of importing it in Unity 2018?

Reply:
Everywhere there is a Text component, I had to add the TextTranslator script. Besides, I had to change the content of "show changelog" to "changelog" because the content in German does not fit.

Reply:
The PatcherLanguages script was in Gameobject Patcher, but it has now been changed to a static class